docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -76,7 +76,8 @@ else:
 
   * **Description**: *auto_pad* has the same definition as *auto_pad* for a regular Convolution but applied in the backward way, for the output tensor. 
     * *explicit*: use explicit padding values from `pads_begin` and `pads_end`.
-    * *same_upper (same_lower)* the input is padded to match the output size. In case of odd padding value an extra padding is added at the end (at the beginning).
+    * *same_upper* the input is padded to match the output size. In case of odd padding value an extra padding is added at the end.
+    * *same_lower* the input is padded to match the output size. In case of odd padding value an extra padding is added at the beginning.
     * *valid* - do not use padding.
   * **Type**: string
```

Comment:
Please add backticks ("`") also for types in all attributes. ie: `**Type**: `string``

Reply:
Fixed.